code-challenges.test.js

```diff
@@ -39,12 +69,49 @@ var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 
 
 // b) Create the function that makes the test pass.
-
-
+                    //PSUEDO
+// create variables to filter the numbers and odds
+//create a function that will but each number from least to greatest
+//return array
+
+let fixFullArr1 = fullArr1.filter(value =>{
+    return typeof value === "number"
+    })
+let fixFullArr2 = fullArr2.filter(value =>{
+        return typeof value === "number"
+        })
+let finalArray = fixFullArr1.filter( value => {
+    return value %2 !== 0
+})
+let finalArray2 = fixFullArr2.filter( value => {
+    return value %2 !== 0
+})
+    const oddsLG = (array) => {
+      let num = 0;
+      for (var i = 0; i < array.length; i++) {
+        for (var j = i; j < array.length; j++) {
+          if (array[j] < array[i]) {
+            num = array[j];
+            array[j] = array[i];
+            array[i] = num;
+          }
+        }
+      }
+      return array;
+    }
+    console.log(oddsLG(finalArray))
+    console.log(oddsLG(finalArray2))
```

Comment:
You need a single function that wraps around all of your code logic that takes in the given inputs dynamically (like what you have on 89). You should not have any code or logic that isn't wrapped in a function that is invoked by the test.